pkg/defaults/defaults.go

```diff
@@ -430,4 +430,8 @@ const (
 	// ExternalClusterIP enables cluster external access to ClusterIP services.
 	// Defaults to false to retain prior behaviour of not routing external packets to ClusterIPs.
 	ExternalClusterIP = false
+
+	// EnableICMPRules enables ICMP/ICMPv6 ciilum network policy rules
+	// todo: remove this after the kernel complexity issue is resolved
```

Comment:
I think we can remove this notes from the PR. Instead, after the PR is merged, let's open a new issue and reference this PR in it.